lvmetrics/server.go

```diff
@@ -0,0 +1,55 @@
+package lvmetrics
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"sync/atomic"
+
+	"github.com/cybozu-go/log"
+	"github.com/cybozu-go/well"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+type logger struct{}
+
+func (l logger) Println(v ...interface{}) {
+	log.Error(fmt.Sprint(v...), nil)
+}
+
+// PromMetricsServer is the Prometheus exporter for lvmetrics
+type PromMetricsServer struct {
+	addr      string
+	collector *Collector
+}
+
+// NewPromMetricsServer constructs metrics exporter of lvmetrics
+func NewPromMetricsServer(addr string, storage *atomic.Value) PromMetricsServer {
+	collector := NewCollector(storage)
+	return PromMetricsServer{addr, collector}
+}
+
+// Run starts the server
+func (p PromMetricsServer) Run(ctx context.Context) error {
+	registry := prometheus.NewRegistry()
+	err := registry.Register(p.collector)
+	if err != nil {
+		return err
+	}
+
+	handler := promhttp.HandlerFor(registry,
+		promhttp.HandlerOpts{
+			ErrorLog:      logger{},
```

Comment:
cybozu-go/well redirect's Go's standrd `log` package to `cybozu-go/log`.
Is this line still necessary?